### DISCUSSION TITLE
Save more stuff in .mlp project files

### DISCUSSION THREAD

**Discussion by elenzil** [cnr-isti-vclab/meshlab#1144] [Ideas]
Hello !

Meshlab's .mlp Project files have been a huge help to me.
I've used them to visually check my matrix math for mesh-alignment in situations where it would have been very difficult to check it otherwise.

I do wish there were a way to save more meta-data so that I could reproduce my views and share them with others tho.
eg
- Window Splitting state
- Trackball state (per window and linked-ness)
- Mesh rendering options (point size, color, shaders, etc)
- Mesh visibility (per window)

Also it would be lovely if project files were in some low-overhead human-readable format like JSON instead of the high-overhead format of XML.

cheers!
Orion

**Comment by alemuntoni** [comment]
Added feature request in #1145. 
Can't say if and when we'll be able to make this available, but we'll keep it as a reminder :) Thanks for the suggestion!
